src/components/image/index.tsx

```diff
@@ -1,8 +1,28 @@
-import { ReactElement } from 'react'
+import { CSSProperties, ReactElement } from 'react'
 import { ImageProps } from './types'
 import classNames from 'classnames'
 import s from './image.module.css'
 
+/**
+ * Create an object to be passed as a style prop to the underlying img element
+ */
+function makeStyleProp(
+  width: ImageProps['width'],
+  height: ImageProps['height']
+): Pick<CSSProperties, 'width' | 'height'> {
+  const style: Pick<CSSProperties, 'width' | 'height'> = {}
+
+  if (width) {
+    style.width = `${width}px`
```

Comment:
q: do we want to support specifying dimensions in non `px` units? We could get that for free by reusing the `HTMLImageElement`'s types.

Reply:
We certainly could, I was initially basing it off of the fact that the height/width attributes only support integer values: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/img#attr-height

Reply:
Personally, I think `style` gives far too much flexibility. We have a general plan to create image guidelines (for example: don't add padding within an image when creating it, because the site will already add margin/padding via CSS), and the `style` prop would enable people to break the guidelines easily.

I think we should add in props one at a time like we have in this PR, and wait to see what kinds of requests are made for customizing image styling. If we end up seeing a lot of requests for the same thing, then that seems like the time to reassess what props are available.